### ISSUE TITLE
Imported AKS cluster does not show any workload level metrics

### ISSUE THREAD

**Issue by jls-tschanzc** [rancher/rancher#27550]
**What kind of request is this (question/bug/enhancement/feature request):**

bug

**Steps to reproduce (least amount of steps as possible):**

1. Import an AKS Cluster
2. Enable Monitoring (0.0.7 or 0.1.0)
3. Wait a while
4. Go to Project -> Namespace -> Workload (Or browse to any workload/pod metrics in Grafana)
5. Open the "Workload Metrics"

**Result:**

It always states "Not enough data for graph". In Grafana it only shows requests/limits and no CPU/Memory data.

**Other details that may be helpful:**

It has already been this way on Rancher 2.3.5 / Monitoring 0.0.7 and remained the same after upgrading to Rancher 2.4.4 / Monitoring 0.1.0.

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): 2.4.4
- Installation option (single install/HA): HA

**Cluster information**

- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): AKS Imported
- Machine type (cloud/VM/metal) and specifications (CPU/memory): Cloud, 1.9 vCPU / 4.5 GB
- Kubernetes version (use `kubectl version`): v1.15.10

```
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.2", GitCommit:"52c56ce7a8272c798dbc29846288d7cd9fbae032", GitTreeState:"clean", BuildDate:"2020-04-16T23:34:25Z", GoVersion:"go1.14.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.10", GitCommit:"059c666b8d0cce7219d2958e6ecc3198072de9bc", GitTreeState:"clean", BuildDate:"2020-04-03T15:17:29Z", GoVersion:"go1.12.12", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by Rancheroo** [comment]
I also have the same issue on
Rancher 2.3.3
AKS 1.14.8

Please advise if more info is required.

**Comment by bentastic27** [comment]
I'm seeing the same behavior

Rancher 2.4.8
AKS 1.17.11
Monitoring 0.1.2

Some logs from the metrics-server include the following, so perhaps related.

```
E1110 14:30:11.289027       1 reststorage.go:160] unable to fetch pod metrics for pod default/nginx-649c7cb7f9-xqmxg: no metrics known for pod
```

**Comment by bentastic27** [comment]
This doesn't seem to work with monitoring 0.1.4 either.

**Comment by GGGitBoy** [comment]
I tried to reproduce these steps and found this error.
If your case is the same as mine. Then the following may be helpful to you.

![______2020-11-24______7 06 33的副本](https://user-images.githubusercontent.com/51182588/100110807-5af01900-2ea8-11eb-87e8-c75dac34f0f7.png)

View the kubelet process of the aks node is the default port 10250, but this is different from the port we scraped

<img width="790" alt="截屏2020-11-24 下午7 07 30" src="https://user-images.githubusercontent.com/51182588/100110923-7ce99b80-2ea8-11eb-8c13-fcc2176ebcf4.png">

### Why is it like this:

After enabling monitoring for some imported clusters, ui will set `exporter-kubelets.https` to `false`

![______2020-11-24______7 12 40的副本](https://user-images.githubusercontent.com/51182588/100110996-92f75c00-2ea8-11eb-9e52-25c4e2455c39.png)

So our scraping port will be 10255 instead of 10250, for aks cluster this is not what we expected

### Workaround:

1. Add monitoring answer `exporter-kubelets.https: true`, for example:

![______2020-11-24______7 21 59的副本](https://user-images.githubusercontent.com/51182588/100111060-a9051c80-2ea8-11eb-91c9-1e6dd54a72c8.png)

2. Save and wait for the monitoring deployment to complete

now it works fine

![______2020-11-24______7 19 05的副本](https://user-images.githubusercontent.com/51182588/100111101-b3271b00-2ea8-11eb-9b81-5fd68898e13d.png)

**Comment by bentastic27** [comment]
@GGGitBoy that is a solid workaround for the time being. The downside is that answer disappears from the monitoring config page after you hit save so you have to remember to re-add it on every change.

**Comment by GGGitBoy** [comment]
yep, this is because UI sends the incorrect `exporter-kubelets.https` parameter according to the cluster type (aks). it will be fixed later
